### ISSUE TITLE
Remove tab order customisations in CMS admin

### ISSUE THREAD

**Issue by thibaudcolas** [wagtail/wagtail#5351]
Some parts of the Wagtail UI define a customisation of the order in which items on the page can be tabbed-through with the `tabindex` attribute. This is generally very confusing for end users, especially when they rely on the keyboard to navigate through the page, and should be removed.

An example of where this happens is the page editing actions’ dropdown, where “Publish” is before “Unpublish”:

<img width="702" alt="tabindex" src="https://user-images.githubusercontent.com/877585/58895829-bd6dbc00-86ec-11e9-9d89-8a79c0fcb1fe.png">

---

The solution for this is to remove all code that sets `tabindex` values greater than 0: https://github.com/wagtail/wagtail/search?q=tabindex&unscoped_q=tabindex

**Comment by thibaudcolas** [comment]
Fixed by #5383.
